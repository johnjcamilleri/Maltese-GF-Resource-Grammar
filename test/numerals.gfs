-- Treebank numerals test
-- Run using the following command:
-- ./_gf < test/numerals.gfs > test/numerals.out

se utf8
i -retain AllMlt.gf

cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 pot01))) -- 1
cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 (pot0 n2)))) -- 2
cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 (pot0 n3)))) -- 3
cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 (pot0 n4)))) -- 4
cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 (pot0 n5)))) -- 5
cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 (pot0 n6)))) -- 6
cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 (pot0 n7)))) -- 7
cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 (pot0 n8)))) -- 8
cc -table -unqual num (pot2as3 (pot1as2 (pot0as1 (pot0 n9)))) -- 9
cc -table -unqual num (pot2as3 (pot1as2 pot110)) -- 10
cc -table -unqual num (pot2as3 (pot1as2 pot111)) -- 11
cc -table -unqual num (pot2as3 (pot1as2 (pot1to19 n3))) -- 13
cc -table -unqual num (pot2as3 (pot1as2 (pot1to19 n4))) -- 14
cc -table -unqual num (pot2as3 (pot1as2 (pot1 n7))) -- 70
cc -table -unqual num (pot2as3 (pot2 pot01)) -- 100
cc -table -unqual num (pot2as3 (pot2plus pot01 (pot1to19 n2))) -- 112
cc -table -unqual num (pot2as3 (pot2plus pot01 (pot1to19 n4))) -- 114
cc -table -unqual num (pot2as3 (pot2 (pot0 n4))) -- 400
cc -table -unqual num (pot2as3 (pot2plus (pot0 n4) pot110)) -- 410
cc -table -unqual num (pot2as3 (pot2plus (pot0 n5) (pot0as1 pot01))) -- 501
cc -table -unqual num (pot2as3 (pot2 (pot0 n6))) -- 600
cc -table -unqual num (pot2as3 (pot2 (pot0 n8))) -- 800
cc -table -unqual num (pot2as3 (pot2plus (pot0 n9) pot111)) -- 911
cc -table -unqual num (pot3 (pot1as2 (pot0as1 pot01))) -- 1,000
cc -table -unqual num (pot3 (pot1as2 pot111)) -- 11,000
cc -table -unqual num (pot3plus (pot1as2 pot111) (pot1as2 (pot1to19 n2))) -- 11,012
cc -table -unqual num (pot3 (pot1as2 (pot1 n4))) -- 40,000
cc -table -unqual num (pot3plus (pot1as2 (pot1plus n9 pot01)) (pot2 (pot0 n7))) -- 91,700
cc -table -unqual num (pot3 (pot2 pot01)) -- 100,000
cc -table -unqual num (pot3plus (pot2 pot01) (pot2 pot01)) -- 100,100
cc -table -unqual num (pot3plus (pot2 pot01) (pot1as2 (pot1plus n3 pot01))) 100,031
cc -table -unqual num (pot3 (pot2plus pot01 pot110)) -- 110,000
cc -table -unqual num (pot3 (pot2plus pot01 pot111)) -- 111,000
cc -table -unqual num (pot3plus (pot2plus pot01 (pot1plus n2 pot01)) (pot2 pot01)) -- 121,100
cc -table -unqual num (pot3 (pot2plus pot01 (pot1 n3))) -- 130,000
cc -table -unqual num (pot3 (pot2plus pot01 (pot1plus n4 pot01))) -- 141,000
cc -table -unqual num (pot3 (pot2plus pot01 (pot1 n6))) -- 160,000
cc -table -unqual num (pot3 (pot2 (pot0 n2))) -- 200,000
cc -table -unqual num (pot3plus (pot2 (pot0 n2)) (pot2plus (pot0 n4) pot110)) -- 200,410
cc -table -unqual num (pot3 (pot2 (pot0 n6))) -- 600,000
cc -table -unqual num (pot3plus (pot2plus (pot0 n6) pot110) (pot1as2 pot110)) -- 610,010
cc -table -unqual num (pot3plus (pot2 (pot0 n7)) (pot2plus (pot0 n6) pot110)) -- 700,610
cc -table -unqual num (pot3plus (pot2plus (pot0 n8) (pot1 n5)) (pot1as2 (pot1to19 n7))) -- 850,017
cc -table -unqual num (pot3plus (pot2 (pot0 n9)) (pot2plus pot01 pot110)) -- 900,110
cc -table -unqual num (pot3plus (pot2 (pot0 n9)) (pot2 (pot0 n2))) -- 900,200
