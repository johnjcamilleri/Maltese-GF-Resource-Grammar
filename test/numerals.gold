s NCard NumNominative : wieħeds NCard NumAdjectival : wieħeds NOrd NumNominative : l- ewwels NOrd NumAdjectival : l- ewwels NCard NumNominative : tnejns NCard NumAdjectival : żewġs NOrd NumNominative : it- tienis NOrd NumAdjectival : it- tienis NCard NumNominative : tlietas NCard NumAdjectival : tletts NOrd NumNominative : it- tielets NOrd NumAdjectival : it- tielets NCard NumNominative : erbgħas NCard NumAdjectival : erba's NOrd NumNominative : ir- raba's NOrd NumAdjectival : ir- raba's NCard NumNominative : ħamsas NCard NumAdjectival : ħamess NOrd NumNominative : il- ħamess NOrd NumAdjectival : il- ħamess NCard NumNominative : sittas NCard NumAdjectival : sitts NOrd NumNominative : is- sitts NOrd NumAdjectival : is- sitts NCard NumNominative : sebgħas NCard NumAdjectival : seba's NOrd NumNominative : is- seba's NOrd NumAdjectival : is- seba's NCard NumNominative : tmienjas NCard NumAdjectival : tmins NOrd NumNominative : it- tmins NOrd NumAdjectival : it- tmins NCard NumNominative : disgħas NCard NumAdjectival : disa's NOrd NumNominative : id- disa's NOrd NumAdjectival : id- disa's NCard NumNominative : għaxras NCard NumAdjectival : għaxars NOrd NumNominative : l- għaxars NOrd NumAdjectival : l- għaxars NCard NumNominative : ħdaxs NCard NumAdjectival : ħdax-ils NOrd NumNominative : il- ħdax-ils NOrd NumAdjectival : il- ħdax-ils NCard NumNominative : tlettaxs NCard NumAdjectival : tlettax-ils NOrd NumNominative : it- tlettax-ils NOrd NumAdjectival : it- tlettax-ils NCard NumNominative : erbataxs NCard NumAdjectival : erbatax-ils NOrd NumNominative : l- erbatax-ils NOrd NumAdjectival : l- erbatax-ils NCard NumNominative : sebgħins NCard NumAdjectival : sebgħins NOrd NumNominative : is- sebgħins NOrd NumAdjectival : is- sebgħins NCard NumNominative : mijas NCard NumAdjectival : mitts NOrd NumNominative : il- mitts NOrd NumAdjectival : il- mitts NCard NumNominative : mija u tnaxs NCard NumAdjectival : mija u tnax-ils NOrd NumNominative : il- mija u tnaxs NOrd NumAdjectival : il- mija u tnax-ils NCard NumNominative : mija u erbataxs NCard NumAdjectival : mija u erbatax-ils NOrd NumNominative : il- mija u erbataxs NOrd NumAdjectival : il- mija u erbatax-ils NCard NumNominative : erba' mijas NCard NumAdjectival : erba' mitts NOrd NumNominative : l- erba' mitts NOrd NumAdjectival : l- erba' mitts NCard NumNominative : erba' mija u għaxras NCard NumAdjectival : erba' mija u għaxars NOrd NumNominative : l- erba' mija u għaxras NOrd NumAdjectival : l- erba' mija u għaxars NCard NumNominative : ħames mija u wieħeds NCard NumAdjectival : ħames mija u wieħeds NOrd NumNominative : il- ħames mija u wieħeds NOrd NumAdjectival : il- ħames mija u wieħeds NCard NumNominative : sitt mijas NCard NumAdjectival : sitt mitts NOrd NumNominative : is- sitt mitts NOrd NumAdjectival : is- sitt mitts NCard NumNominative : tmin mijas NCard NumAdjectival : tmin mitts NOrd NumNominative : it- tmin mitts NOrd NumAdjectival : it- tmin mitts NCard NumNominative : disa' mija u ħdaxs NCard NumAdjectival : disa' mija u ħdax-ils NOrd NumNominative : id- disa' mija u ħdaxs NOrd NumAdjectival : id- disa' mija u ħdax-il44 msecNumeral> See you.