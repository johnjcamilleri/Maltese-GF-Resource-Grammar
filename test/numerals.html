<html>
<head>
	<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
	<style type="text/css">
		body { font: 16px/1.5 serif; }
		div { margin-left: 2em; }
		#text-output { margin:0; color:#999; }
		#text-output b { display: inline-block; width: 10em; font-weight:normal; }
		#text-output em { font-style: italic; color:#000; }
		#text-output em.correct, .correct { color:green; }
		#text-output em.incorrect, .incorrect { color:red; }
		#text-output em.gold, .gold { background-color:gold; padding:0 5px; }

		table { border-collapse:collapse; border-spacing:0; }
		tr:nth-child(odd) { background-color:#eee; }
		tr.separator { background-color:#000; font-size:2px; }
		td { padding:0.2em 1em; border-width:1px 0; border-style:solid; border-color:#ccc; }
		td:last-child { font-style:italic; }

	</style>
</head>
<body>

<h1>Tree linearisations for: numerals</h1>
<p><em>Generated Thu, 25 Aug 2011 22:28:56 +0200</em></p>
<p><span class="correct">82 correct</span>, <span class="incorrect">10 incorrect</span></p>
<hr/>

<div id="text-output">
<div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>wieħed</em></div><div><b>NumAdjectival</b><em>wieħed</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- ewwel</em></div><div><b>NumAdjectival</b><em>l- ewwel</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tnejn</em></div><div><b>NumAdjectival</b><em>żewġ</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tieni</em></div><div><b>NumAdjectival</b><em>it- tieni</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tlieta</em></div><div><b>NumAdjectival</b><em>tlett</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tielet</em></div><div><b>NumAdjectival</b><em>it- tielet</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>erbgħa</em></div><div><b>NumAdjectival</b><em>erba'</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>ir- raba'</em></div><div><b>NumAdjectival</b><em>ir- raba'</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>ħamsa</em></div><div><b>NumAdjectival</b><em>ħames</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- ħames</em></div><div><b>NumAdjectival</b><em>il- ħames</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>sitta</em></div><div><b>NumAdjectival</b><em>sitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>is- sitt</em></div><div><b>NumAdjectival</b><em>is- sitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>sebgħa</em></div><div><b>NumAdjectival</b><em>seba'</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>is- seba'</em></div><div><b>NumAdjectival</b><em>is- seba'</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tmienja</em></div><div><b>NumAdjectival</b><em>tmin</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tmin</em></div><div><b>NumAdjectival</b><em>it- tmin</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>disgħa</em></div><div><b>NumAdjectival</b><em>disa'</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>id- disa'</em></div><div><b>NumAdjectival</b><em>id- disa'</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>għaxra</em></div><div><b>NumAdjectival</b><em>għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- għaxar</em></div><div><b>NumAdjectival</b><em>l- għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>ħdax</em></div><div><b>NumAdjectival</b><em>ħdax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- ħdax</em></div><div><b>NumAdjectival</b><em>il- ħdax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tlettax</em></div><div><b>NumAdjectival</b><em>tlettax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tlettax</em></div><div><b>NumAdjectival</b><em>it- tlettax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>erbatax</em></div><div><b>NumAdjectival</b><em>erbatax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- erbatax</em></div><div><b>NumAdjectival</b><em>l- erbatax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>sebgħin</em></div><div><b>NumAdjectival</b><em>sebgħin</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>is- sebgħin</em></div><div><b>NumAdjectival</b><em>is- sebgħin</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija</em></div><div><b>NumAdjectival</b><em>mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija</em></div><div><b>NumAdjectival</b><em>il- mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u tnax</em></div><div><b>NumAdjectival</b><em>mija u tnax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>il- mija u it- tnax</em> &rarr; <em class='gold'>il- mija u tnax</em></div><div><b>NumAdjectival</b><em class='incorrect'>il- mija u it- tnax-il</em> &rarr; <em class='gold'>il- mija u tnax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u erbatax</em></div><div><b>NumAdjectival</b><em>mija u erbatax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>il- mija u l- erbatax</em> &rarr; <em class='gold'>il- mija u erbatax</em></div><div><b>NumAdjectival</b><em class='incorrect'>il- mija u l- erbatax-il</em> &rarr; <em class='gold'>il- mija u erbatax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>erba' mija</em></div><div><b>NumAdjectival</b><em>erba' mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- erba' mija</em></div><div><b>NumAdjectival</b><em>l- erba' mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>erba' mija u għaxra</em></div><div><b>NumAdjectival</b><em>erba' mija u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>l- erba' mija u l- għaxar</em> &rarr; <em class='gold'>l- erba' mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>l- erba' mija u l- għaxar</em> &rarr; <em class='gold'>l- erba' mija u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>ħames mija u wieħed</em></div><div><b>NumAdjectival</b><em>ħames mija u wieħed</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>il- ħames mija u l- ewwel</em> &rarr; <em class='gold'>il- ħames mija u wieħed</em></div><div><b>NumAdjectival</b><em class='incorrect'>il- ħames mija u l- ewwel</em> &rarr; <em class='gold'>il- ħames mija u wieħed</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>sitt mija</em></div><div><b>NumAdjectival</b><em>sitt mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>is- sitt mija</em></div><div><b>NumAdjectival</b><em>is- sitt mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tmin mija</em></div><div><b>NumAdjectival</b><em>tmin mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tmin mija</em></div><div><b>NumAdjectival</b><em>it- tmin mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>disa' mija u ħdax</em></div><div><b>NumAdjectival</b><em>disa' mija u ħdax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>id- disa' mija u il- ħdax</em> &rarr; <em class='gold'>id- disa' mija u ħdax</em></div><div><b>NumAdjectival</b><em class='incorrect'>id- disa' mija u il- ħdax-il</em> &rarr; <em class='gold'>id- disa' mija u ħdax-il</em></div></div></div><hr/>
</div>

</body>
</html>