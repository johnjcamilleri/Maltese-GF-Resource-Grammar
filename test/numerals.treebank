Numeral: num (pot2as3 (pot1as2 (pot0as1 pot01)))
NumeralMlt: wieħed
l-ewwel

 Numeral: num (pot2as3 (pot1as2 (pot0as1 (pot0 n2))))
NumeralMlt: tnejn
tieni

 Numeral: num (pot2as3 (pot1as2 (pot0as1 (pot0 n3))))
NumeralMlt: tlieta
tielet

 Numeral: num (pot2as3 (pot1as2 (pot0as1 (pot0 n4))))
NumeralMlt: erbgħa
raba

 Numeral: num (pot2as3 (pot1as2 (pot0as1 (pot0 n5))))
NumeralMlt: ħamsa
ħames

 Numeral: num (pot2as3 (pot1as2 (pot0as1 (pot0 n6))))
NumeralMlt: sitta
sitt

 Numeral: num (pot2as3 (pot1as2 (pot0as1 (pot0 n7))))
NumeralMlt: sebgħa
seba'

 Numeral: num (pot2as3 (pot1as2 (pot0as1 (pot0 n8))))
NumeralMlt: tmienja
tmin

 Numeral: num (pot2as3 (pot1as2 (pot0as1 (pot0 n9))))
NumeralMlt: disgħa
disa'

 Numeral: num (pot2as3 (pot1as2 pot110))
NumeralMlt: għaxra
għaxra

 Numeral: num (pot2as3 (pot1as2 pot111))
NumeralMlt: ħdax
ħdax

 Numeral: num (pot2as3 (pot1as2 (pot1to19 n3)))
NumeralMlt: tlettax
tlettax

 Numeral: num (pot2as3 (pot1as2 (pot1to19 n4)))
NumeralMlt: erbatax
erbatax

 Numeral: num (pot2as3 (pot1as2 (pot1 n7)))
NumeralMlt: sebgħin
sebgħin

 Numeral: num (pot2as3 (pot2 pot01))
NumeralMlt: mija
mija

 Numeral: num (pot2as3 (pot2plus pot01 (pot1to19 n2)))
NumeralMlt: mija u tnax
mija u tnax

 Numeral: num (pot2as3 (pot2plus pot01 (pot1to19 n4)))
NumeralMlt: mija u erbatax
mija u erbatax

 Numeral: num (pot2as3 (pot2 (pot0 n4)))
NumeralMlt: erba' mija
erba' mija

 Numeral: num (pot2as3 (pot2plus (pot0 n4) pot110))
NumeralMlt: erba' mija u għaxra
erba' mija u għaxra

 Numeral: num (pot2as3 (pot2plus (pot0 n5) (pot0as1 pot01)))
NumeralMlt: ħames mija u wieħed
ħames mija u wieħed

 Numeral: num (pot2as3 (pot2 (pot0 n6)))
NumeralMlt: sitt mija
sitt mija

 Numeral: num (pot2as3 (pot2 (pot0 n8)))
NumeralMlt: tmien mija
tmien mija

 Numeral: num (pot2as3 (pot2plus (pot0 n9) pot111))
NumeralMlt: disa' mija u ħdax
disa' mija u ħdax

 Numeral: num (pot3 (pot1as2 (pot0as1 pot01)))
NumeralMlt: elf
elf

 Numeral: num (pot3 (pot1as2 pot111))
NumeralMlt: ħdax-il elef
ħdax-il elef

 Numeral: num (pot3plus (pot1as2 pot111) (pot1as2 (pot1to19 n2)))
NumeralMlt: ħdax-il elef tnax
ħdax-il elef tnax

 Numeral: num (pot3 (pot1as2 (pot1 n4)))
NumeralMlt: erbgħin elef
erbgħin elef

 Numeral: num (pot3plus (pot1as2 (pot1plus n9 pot01)) (pot2 (pot0 n7)))
NumeralMlt: wieħed u disgħin elef seba' mija
wieħed u disgħin elef seba' mija

 Numeral: num (pot3 (pot2 pot01))
NumeralMlt: mija elef
mija elef

 Numeral: num (pot3plus (pot2 pot01) (pot2 pot01))
NumeralMlt: mija elef mija
mija elef mija

 Numeral: num (pot3 (pot2plus pot01 pot110))
NumeralMlt: mija u għaxra elef
mija u għaxra elef

 Numeral: num (pot3 (pot2plus pot01 pot111))
NumeralMlt: mija u ħdax-il elef
mija u ħdax-il elef

 Numeral: num (pot3plus (pot2plus pot01 (pot1plus n2 pot01)) (pot2 pot01))
NumeralMlt: mija u wieħed u għoxrin elef mija
mija u wieħed u għoxrin elef mija

 Numeral: num (pot3 (pot2plus pot01 (pot1 n3)))
NumeralMlt: mija u tletin elef
mija u tletin elef

 Numeral: num (pot3 (pot2plus pot01 (pot1plus n4 pot01)))
NumeralMlt: mija u wieħed u erbgħin elef
mija u wieħed u erbgħin elef

 Numeral: num (pot3 (pot2plus pot01 (pot1 n6)))
NumeralMlt: mija u sitt elef
mija u sitt elef

 Numeral: num (pot3 (pot2 (pot0 n2)))
NumeralMlt: mitejn elef
mitejn elef

 Numeral: num (pot3plus (pot2 (pot0 n2)) (pot2plus (pot0 n4) pot110))
NumeralMlt: mitejn elef erba' mija u għaxra
mitejn elef erba' mija u għaxra

 Numeral: num (pot3 (pot2 (pot0 n6)))
NumeralMlt: sitt mija elef
sitt mija elef

 Numeral: num (pot3plus (pot2plus (pot0 n6) pot110) (pot1as2 pot110))
NumeralMlt: sitt mija u għaxra elef għaxra
sitt mija u għaxra elef għaxra

 Numeral: num (pot3plus (pot2 (pot0 n7)) (pot2plus (pot0 n6) pot110))
NumeralMlt: seba' mija elef sitt mija u għaxra
seba' mija elef sitt mija u għaxra

 Numeral: num (pot3plus (pot2plus (pot0 n8) (pot1 n5)) (pot1as2 (pot1to19 n7)))
NumeralMlt: tmien mija u ħamsin elef sbatax
tmien mija u ħamsin elef sbatax

 Numeral: num (pot3plus (pot2 (pot0 n9)) (pot2plus pot01 pot110))
NumeralMlt: disa' mija elef mija u għaxra
disa' mija elef mija u għaxra

 Numeral: num (pot3plus (pot2 (pot0 n9)) (pot2 (pot0 n2)))
NumeralMlt: disa' mija elef mitejn
disa' mija elef mitejn

