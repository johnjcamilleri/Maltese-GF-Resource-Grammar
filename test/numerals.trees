-- Run with:
-- rf -lines -tree -file=test/numerals.trees | l -table

-- 1
num (pot2as3 (pot1as2 (pot0as1 pot01)))

-- 2
num (pot2as3 (pot1as2 (pot0as1 (pot0 n2))))

-- 3
num (pot2as3 (pot1as2 (pot0as1 (pot0 n3))))

-- 4
num (pot2as3 (pot1as2 (pot0as1 (pot0 n4))))

-- 5
num (pot2as3 (pot1as2 (pot0as1 (pot0 n5))))

-- 6
num (pot2as3 (pot1as2 (pot0as1 (pot0 n6))))

-- 7
num (pot2as3 (pot1as2 (pot0as1 (pot0 n7))))

-- 8
num (pot2as3 (pot1as2 (pot0as1 (pot0 n8))))

-- 9
num (pot2as3 (pot1as2 (pot0as1 (pot0 n9))))

-- 10
num (pot2as3 (pot1as2 pot110))

-- 11
num (pot2as3 (pot1as2 pot111))

-- 13
num (pot2as3 (pot1as2 (pot1to19 n3)))

-- 14
num (pot2as3 (pot1as2 (pot1to19 n4)))

-- 70
num (pot2as3 (pot1as2 (pot1 n7)))

-- 100
num (pot2as3 (pot2 pot01))

-- 112
num (pot2as3 (pot2plus pot01 (pot1to19 n2)))

-- 114
num (pot2as3 (pot2plus pot01 (pot1to19 n4)))

-- 400
num (pot2as3 (pot2 (pot0 n4)))

-- 410
num (pot2as3 (pot2plus (pot0 n4) pot110))

-- 501
num (pot2as3 (pot2plus (pot0 n5) (pot0as1 pot01)))

-- 600
num (pot2as3 (pot2 (pot0 n6)))

-- 800
num (pot2as3 (pot2 (pot0 n8)))

-- 911
num (pot2as3 (pot2plus (pot0 n9) pot111))

-- 1,000
num (pot3 (pot1as2 (pot0as1 pot01)))

-- 11,000
num (pot3 (pot1as2 pot111))

-- 11,012
num (pot3plus (pot1as2 pot111) (pot1as2 (pot1to19 n2)))

-- 40,000
num (pot3 (pot1as2 (pot1 n4)))

-- 91,700
num (pot3plus (pot1as2 (pot1plus n9 pot01)) (pot2 (pot0 n7)))

-- 100,000
num (pot3 (pot2 pot01))

-- 100,100
num (pot3plus (pot2 pot01) (pot2 pot01))

-- 100,031
num (pot3plus (pot2 pot01) (pot1as2 (pot1plus n3 pot01)))

-- 110,000
num (pot3 (pot2plus pot01 pot110))

-- 111,000
num (pot3 (pot2plus pot01 pot111))

-- 121,100
num (pot3plus (pot2plus pot01 (pot1plus n2 pot01)) (pot2 pot01))

-- 130,000
num (pot3 (pot2plus pot01 (pot1 n3)))

-- 141,000
num (pot3 (pot2plus pot01 (pot1plus n4 pot01)))

-- 160,000
num (pot3 (pot2plus pot01 (pot1 n6)))

-- 200,000
num (pot3 (pot2 (pot0 n2)))

-- 200,410
num (pot3plus (pot2 (pot0 n2)) (pot2plus (pot0 n4) pot110))

-- 600,000
num (pot3 (pot2 (pot0 n6)))

-- 610,010
num (pot3plus (pot2plus (pot0 n6) pot110) (pot1as2 pot110))

-- 700,610
num (pot3plus (pot2 (pot0 n7)) (pot2plus (pot0 n6) pot110))

-- 850,017
num (pot3plus (pot2plus (pot0 n8) (pot1 n5)) (pot1as2 (pot1to19 n7)))

-- 900,110
num (pot3plus (pot2 (pot0 n9)) (pot2plus pot01 pot110))

-- 900,200
num (pot3plus (pot2 (pot0 n9)) (pot2 (pot0 n2)))
