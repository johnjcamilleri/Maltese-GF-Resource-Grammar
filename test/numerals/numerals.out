s NCard NumNominative : wieħed
s NCard NumAdjectival : wieħed
s NOrd NumNominative : l- &+ ewwel
s NOrd NumAdjectival : l- &+ ewwel


s NCard NumNominative : tnejn
s NCard NumAdjectival : żewġ
s NOrd NumNominative : it- &+ tieni
s NOrd NumAdjectival : it- &+ tieni


s NCard NumNominative : tlieta
s NCard NumAdjectival : tlett
s NOrd NumNominative : it- &+ tielet
s NOrd NumAdjectival : it- &+ tielet


s NCard NumNominative : erbgħa
s NCard NumAdjectival : erba'
s NOrd NumNominative : ir- &+ raba'
s NOrd NumAdjectival : ir- &+ raba'


s NCard NumNominative : ħamsa
s NCard NumAdjectival : ħames
s NOrd NumNominative : il- &+ ħames
s NOrd NumAdjectival : il- &+ ħames


s NCard NumNominative : sitta
s NCard NumAdjectival : sitt
s NOrd NumNominative : is- &+ sitt
s NOrd NumAdjectival : is- &+ sitt


s NCard NumNominative : sebgħa
s NCard NumAdjectival : seba'
s NOrd NumNominative : is- &+ seba'
s NOrd NumAdjectival : is- &+ seba'


s NCard NumNominative : tmienja
s NCard NumAdjectival : tmin
s NOrd NumNominative : it- &+ tmin
s NOrd NumAdjectival : it- &+ tmin


s NCard NumNominative : disgħa
s NCard NumAdjectival : disa'
s NOrd NumNominative : id- &+ disa'
s NOrd NumAdjectival : id- &+ disa'


s NCard NumNominative : għaxra
s NCard NumAdjectival : għaxar
s NOrd NumNominative : l- &+ għaxar
s NOrd NumAdjectival : l- &+ għaxar


s NCard NumNominative : ħdax
s NCard NumAdjectival : ħdax-il
s NOrd NumNominative : il- &+ ħdax
s NOrd NumAdjectival : il- &+ ħdax-il


s NCard NumNominative : tlettax
s NCard NumAdjectival : tlettax-il
s NOrd NumNominative : it- &+ tlettax
s NOrd NumAdjectival : it- &+ tlettax-il


s NCard NumNominative : erbatax
s NCard NumAdjectival : erbatax-il
s NOrd NumNominative : l- &+ erbatax
s NOrd NumAdjectival : l- &+ erbatax-il


s NCard NumNominative : sebgħin
s NCard NumAdjectival : sebgħin
s NOrd NumNominative : is- &+ sebgħin
s NOrd NumAdjectival : is- &+ sebgħin


s NCard NumNominative : mija
s NCard NumAdjectival : mitt
s NOrd NumNominative : il- &+ mija
s NOrd NumAdjectival : il- &+ mitt


s NCard NumNominative : mija u tnax
s NCard NumAdjectival : mija u tnax-il
s NOrd NumNominative : il- &+ mija u tnax
s NOrd NumAdjectival : il- &+ mija u tnax-il


s NCard NumNominative : mija u erbatax
s NCard NumAdjectival : mija u erbatax-il
s NOrd NumNominative : il- &+ mija u erbatax
s NOrd NumAdjectival : il- &+ mija u erbatax-il


s NCard NumNominative : erba' mija
s NCard NumAdjectival : erba' mitt
s NOrd NumNominative : l- &+ erba' mija
s NOrd NumAdjectival : l- &+ erba' mitt


s NCard NumNominative : erba' mija u għaxra
s NCard NumAdjectival : erba' mija u għaxar
s NOrd NumNominative : l- &+ erba' mija u għaxra
s NOrd NumAdjectival : l- &+ erba' mija u għaxar


s NCard NumNominative : ħames mija u wieħed
s NCard NumAdjectival : ħames mija u wieħed
s NOrd NumNominative : il- &+ ħames mija u wieħed
s NOrd NumAdjectival : il- &+ ħames mija u wieħed


s NCard NumNominative : sitt mija
s NCard NumAdjectival : sitt mitt
s NOrd NumNominative : is- &+ sitt mija
s NOrd NumAdjectival : is- &+ sitt mitt


s NCard NumNominative : tmin mija
s NCard NumAdjectival : tmin mitt
s NOrd NumNominative : it- &+ tmin mija
s NOrd NumAdjectival : it- &+ tmin mitt


s NCard NumNominative : disa' mija u ħdax
s NCard NumAdjectival : disa' mija u ħdax-il
s NOrd NumNominative : id- &+ disa' mija u ħdax
s NOrd NumAdjectival : id- &+ disa' mija u ħdax-il



88 msec
AllMltAbs> See you.
0 msec